pkg/quota/v1/evaluator/core/pods.go

```diff
@@ -353,10 +353,17 @@ func podMatchesScopeFunc(selector corev1.ScopedResourceSelectorRequirement, obje
 		return false, err
 	}
 	switch selector.ScopeName {
+	// Fixed: Now correctly checks deletionTimestamp instead of activeDeadlineSeconds
+	// See: https://github.com/kubernetes/kubernetes/issues/135411
```

Comment:
I'm not sure we should redefine this to *not* include pods with set deadlines... *expanding* it to include pods with deletiontimestamp set seems more reasonable